vortex-api/src/main/java/mil/army/usace/hec/vortex/io/Hdf5RasPrecipDataWriter.java

```diff
@@ -258,5 +259,21 @@ private void closer(HdfCloser hc){
                 logger.severe("Failed to close hdf resource:" + ex.getMessage());
             }
         }
+
+        private float[] verticalTranpose(VortexGrid grid) {
+            float[] data = grid.data();
+            float[] transposedData = new float[data.length];
+            int rows = grid.nx();
+            int cols = grid.ny();
+            for (int r = 0; r < rows; r++) {
```

Comment:
You can call `MatrixUtils::flipArray` here which should do the vertical transpose (and reuse code).